awx/main/analytics/collectors.py

```diff
@@ -419,7 +419,7 @@ def query(event_data):
                           resolved_action,
                           resolved_role,
                           -- '-' operator listed here:
-                          -- https://www.postgresql.org/docs/12/functions-json.html
+                          -- https://www.postgresql.org/docs/15/functions-json.html
                           -- note that operator is only supported by jsonb objects
                           -- https://www.postgresql.org/docs/current/datatype-json.html
```

Comment:
@john-westcott-iv can we remove this?

Reply:
I believe they are comments so we /could/ remove them if we wanted but I'd say leave them.